manifests/publisher.yaml

```
@@ -0,0 +1,43 @@
+        command:
+        - /usr/local/bin/main
+        - start
+        - publiser
+        - --log-level=debug
+        - --log-headers=true
+        - --log-bodies=true
+        - --db-url=postgres://service:service123@fulfillment-database:5432/service
+        ports:
+        - name: metrics
+          protocol: TCP
+          containerPort: 8001
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)